CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Feature: Support for `BlitzBasic`. ([@Lukaszeq](https://github.com/Lukaszeq) in [#2609](https://github.com/vscode-icons/vscode-icons/pull/2609))
 - Enhancement: Add `BUILD.bazel` extension and update language id. ([@CaringDev](https://github.com/CaringDev) in [#2605](https://github.com/vscode-icons/vscode-icons/pull/2605))
 - Feature: Support for `Hook(s)` folder. ([@chancegraff](https://github.com/chancegraff) in [#2604](https://github.com/vscode-icons/vscode-icons/pull/2604))
+- Enhancement: Add `cjs` to `rollup`. ([@sant123](https://github.com/sant12) in [#2603](https://github.com/vscode-icons/vscode-icons/pull/2603))
```

Comment:
The correct link is `https://github.com/sant123`

Reply:
solved